website/docs/d/google_billing_account.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "google"
+page_title: "Google: google_billing_account"
+sidebar_current: "docs-google-datasource-billing-account"
+description: |-
+  Get information about a Google Billing Account.
+---
+
+# google\_billing\_account
+
+Use this data source to get information about a Google Billing Account.
+
+```hcl
+data "google_billing_account" "acct" {
+  display_name = "My Billing Account"
+}
+
+resource "google_project" "my_project" {
+  name       = "My Project"
+  project_id = "your-project-id"
+  org_id     = "1234567"
+
+  billing_account = "${data.google_billing_account.acct.id}"
+}
+```
+
+## Argument Reference
+
+The arguments of this data source act as filters for querying the available billing accounts.
+The given filters must match exactly one billing account whose data will be exported as attributes.
+The following arguments are supported:
+
+* `name` (Optional) - The resource name of the billing account in the form `billingAccounts/{billing_account_id}`.
+* `display_name` (Optional) - The display name of the billing account.
+
+~> **NOTE:** One of `name` or `display_name` must be specified.
+
+## Attributes Reference
+
+The following additional attributes are exported:
+
+* `id` - The billing account ID.
+* `open` - `true` if the billing account is open, `false` if the billing account is closed.
```

Comment:
Do you think filtering by "open = true" might be useful?

If you think so, you can define this field both Computed and Optional. This means that if someone specifies it, it will filter, if they don't specify it, you can still set the value from the server.

Reply:
Yes, makes sense,

Reply:
Done.